src/WebJobs.Script/Description/DotNet/DependencyHelper.cs

```diff
@@ -165,5 +166,33 @@ public static List<string> GetRuntimeFallbacks(string rid)
             rids.AddRange(fallbacks.Fallbacks);
             return rids;
         }
+
+        /// <summary>
+        /// Checks if the string is in shared assembly representation format.
+        /// </summary>
+        /// <param name="assemblyFormatString"> string representing assembly information</param>
+        /// <returns> bool if string in shared assembly representation format. </returns>
+        public static bool IsSharedAssemblyFormat(string assemblyFormatString)
```

Comment:
Add unit tests for this and all other new methods. There's already a DendencyHelperTests suite